Sources/SwiftProtobuf/Enum.swift

```diff
@@ -17,77 +17,76 @@
-  /// the value that was originally decoded.
-  var rawValue: Int { get }
+      /// The raw integer value of the enum value.
+      ///
+      /// For a recognized enum case, this is the integer value of the case as
+      /// defined in the .proto file. For `UNRECOGNIZED` cases in proto3, this is
+      /// the value that was originally decoded.
+      var rawValue: Int { get }
 }
 
 extension Enum {
-#if swift(>=4.2)
-  public func hash(into hasher: inout Hasher) {
-    hasher.combine(rawValue)
-  }
-#else  // swift(>=4.2)
-  public var hashValue: Int {
-    return rawValue
-  }
-#endif  // swift(>=4.2)
+      #if swift(>=4.2)
+        public func hash(into hasher: inout Hasher) {
+          hasher.combine(rawValue)
+        }
+      #else  // swift(>=4.2)
+        public var hashValue: Int {
+          return rawValue
+        }
+      #endif
 
-  /// Internal convenience property representing the name of the enum value (or
-  /// `nil` if it is an `UNRECOGNIZED` value or doesn't provide names).
-  ///
-  /// Since the text format and JSON names are always identical, we don't need
-  /// to distinguish them.
-  internal var name: _NameMap.Name? {
-    guard let nameProviding = Self.self as? _ProtoNameProviding.Type else {
-      return nil
-    }
-    return nameProviding._protobuf_nameMap.names(for: rawValue)?.proto
-  }
+      /// Internal convenience property representing the name of the enum value (or
+      /// `nil` if it is an `UNRECOGNIZED` value or doesn't provide names).
+      ///
+      /// Since the text format and JSON names are always identical, we don't need
+      /// to distinguish them.
+      internal var name: _NameMap.Name? {
+            guard let nameProviding = Self.self as? _ProtoNameProviding.Type
+            else { return nil }
+            return nameProviding._protobuf_nameMap.names(for: rawValue)?.proto
+      }
 
-  /// Internal convenience initializer that returns the enum value with the
-  /// given name, if it provides names.
-  ///
-  /// Since the text format and JSON names are always identical, we don't need
-  /// to distinguish them.
-  ///
-  /// - Parameter name: The name of the enum case.
-  internal init?(name: String) {
-    guard let nameProviding = Self.self as? _ProtoNameProviding.Type,
-      let number = nameProviding._protobuf_nameMap.number(forJSONName: name) else {
-      return nil
-    }
-    self.init(rawValue: number)
-  }
+      /// Internal convenience initializer that returns the enum value with the
+      /// given name, if it provides names.
+      ///
+      /// Since the text format and JSON names are always identical, we don't need
+      /// to distinguish them.
+      ///
+      /// - Parameter name: The name of the enum case.
+      internal init?(name: String) {
+            guard let nameProviding = Self.self as? _ProtoNameProviding.Type,
+                  let number = nameProviding._protobuf_nameMap.number(
+                  forJSONName: name)
```

Comment:
Another place where the block indentation level (increased by the left paren) and the continuation line state seem to be interacting oddly...

Reply:
Yeah, I believe we got to this point in our analysis and that's when I realized we might need to do something more sophisticated, like reset the nesting->indentation mapping when an unmatched-on-the-same-line close delimiter ends a continuation line, and *that's* when I realized I wanted to stop trying to figure it all out in that PR ;-).  I think the answer fits neatly into your stack-based model, though.